mutt_config.c

```diff
@@ -87,6 +87,20 @@ bool C_IgnoreLinearWhiteSpace = false;
 struct ConfigDef MuttVars[] = {
   /*++*/
 
+  { "abort_key", DT_STRING|DT_NOT_EMPTY, &C_AbortKeyStr, IP "\007" },
+  /*
+  ** .pp
+  ** Specifies the key that can be used to abort prompts.  The format is the
+  ** same as used in "bind" commands.  The default is equivalient to "\G".
+  ** Note that the specified key should not be used in other bindings, as the
+  ** abort operation has higher precedence and the binding will not have the
+  ** desired effect.
+  ** .pp
+  ** Example:
+  ** .ts
+  ** set abort_key = "<Esc>"
+  ** .te
```

Comment:
Please can you add a short note about `ESCDELAY=0`.
What it does, why it's a good idea.

Reply:
Sure thing, see 92e39177c57464c1d51d3adbe8380e9ed35287dd.